pom.xml

```diff
@@ -1069,6 +1069,24 @@
         <version>${hadoop.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-client-runtime</artifactId>
+        <version>${hadoop.version}</version>
+        <scope>provided</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
We should try to do not add exclusions. Are we sure we need to exclude here? Again, dependency management should fix most of the issues we resolved in the past by using exclusions

Reply:
This is an explicitly banned dependency here - https://github.com/apache/pinot/blob/4b5bd217e4a70bc76f60d73897a0484f19955ed5/pom.xml#L1742-L1754

Without this exclusion, the enforcer plugin's enforce goal fails in the build. We're excluding the `commons-logging` dependency from various other dependencies due to this, for example - https://github.com/apache/pinot/blob/4b5bd217e4a70bc76f60d73897a0484f19955ed5/pom.xml#L688-L698